.github/workflows/daily-linkcheck.yml

```
@@ -0,0 +1,21 @@
+name: Daily link check
+on:
+  pull_request:
+  schedule:
+    - cron: '00 19 * * *' # every day at 19:00 UTC
+
+  workflow_dispatch:
+    # Manual trigger
+
+jobs:
+  link-check:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```
Apologies, contrary what I suggested earlier, after some thinking, it seems better to have a stable version to avoid surprising updates and failures.